lib/analyzerinfo.cpp

```diff
@@ -98,6 +98,13 @@ static bool skipAnalysis(const std::string &analyzerInfoFile, std::size_t hash,
     if (!attr || attr != std::to_string(hash))
         return false;
 
+    // Check for invalid license error, in which case we should retry analysis
```

Comment:
This could be tested in a unit test also. but it would require some refactorings.